rocAL/rocAL/include/augmentations/geometry_augmentations/node_rotate.h

```diff
@@ -25,14 +25,15 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_vx.h"
 #include "graph.h"
+#include "rocal_api_types.h"
 
 class RotateNode : public Node
 {
 public:
     RotateNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
     RotateNode() = delete;
-    void init(float angle);
-    void init(FloatParam *angle);
+    void init(float angle, RocalResizeInterpolationType interpolation_type = ROCAL_LINEAR_INTERPOLATION);
```

Comment:
Please check if you need RocalResizeInterpolationType here in rocate.
Remove them if unecessary